src/topology/category/Profinite/default.lean

```diff
@@ -203,6 +203,10 @@ has_limits_of_has_limits_creates_limits Profinite_to_Top
 instance has_colimits : limits.has_colimits Profinite :=
 has_colimits_of_reflective to_CompHaus
 
+noncomputable
+instance forget_preserves_limits : limits.preserves_limits (forget Profinite) :=
```

Comment:
Don't we get this automatically from the fact that the functor creates limits?

Reply:
I think the issue is that the `Profinite_to_Top` is used to create limits, and lean needs to be told explicitly that the composition of this with the forgetful functor on `Top` agrees with the forgetful functor on `Profinite`.

Reply:
Ah I see - should we not add that the functor to Type creates limits as well then?

Reply:
Yes, I agree, but this is a somewhat nontrivial result (since `forget Top` does not create limits), so I would prefer to leave the `preserves_limits` instance for now, and worry about the `creates_limits` instance later on. 

I think the best approach would be to use the fact that `Profinite_to_CompHaus` creates limits, and that `forget CompHaus` is monadic (which involves using the equivalence with `Compactum`).

Reply:
Sounds good!